commit e9aff6b9028b5d537d3918ecddb7808e3c34894b
Author: John Levon <john.levon@joyent.com>
Date:   2019-07-08T10:09:58+00:00 (3 months ago)
    
    joyent/node-zkstream#9 use var instead of const so we can run in strict mode ((foo bar))
    joyent/node-zkstream#29 unreliable tests assuming after sync() all watchers have fired
    joyent/node-zkstream#24 remove watchers from index once all event handlers are gone
    joyent/node-zkstream#30 createWithEmptyParents API tests call Buffer.from
    joyent/node-zkstream#22 increase test coverage

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
